docs/design_docs/011-historacle-pricing.md

```diff
@@ -96,7 +107,7 @@ This will not introduce a new module, and it is relatively backwards compatible.
 
 ## Comments
 
-Originally a 24-hour median was thought to be effective enough to defend against this type of attack, but our modeling shows that we would need 30 days for adequate protection.
+Currently we are planning on keeping the last 6 hours or so of medians, and a longer historic price period. This way each median can be calculated based off of the last 1 hour or 12 hours, and we can perform computations over the last x amount of medians.
```

Comment:
Might clarify what's happening with the 1-12 hours

Reply:
Clarified that the intention is for the "max medians" and "max price stamps" to be agnostic of one another